_rules/SC1-3-1-aria-required-context-role.md

```diff
@@ -0,0 +1,145 @@
+---
+name: ARIA required context role
+description: | 
+  This rule checks that a role does not exist outside of its required context roles
+
+success_criterion:
+- 1.3.1 # Info and Relationships
+
+test_aspects: 
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any HTML or SVG element that is [exposed to assistive technologies](#exposed-to-assistive-technologies) and has an explicit [semantic role](#semantic-role) that has [required context roles](https://www.w3.org/TR/wai-aria-1.1/#scope) listed for that role in [WAI-ARIA](https://www.w3.org/TR/wai-aria).
```

Comment:
- IMO, this doesn't quite pass the "plain language" smell test. Can we do anything about this?
- We should be specific about what ARIA spec we reference. I left a comment earlier about possibly having a definition "current ARIA specifications". Lets discuss.

Reply:
I suggest prefixing or suffixing phrases that are drawn from external sources with the name of these sources. For example, "WAI-ARIA 1.1 required context rules" or "required context rules (WAI-ARIA 1.1)"